tools/cli/wsk

```diff
@@ -100,6 +102,8 @@ def parseArgs(props):
 
     parser.add_argument('--apihost', help='whisk API host', dest='apihostOverride', metavar='hostname')
     parser.add_argument('--apiversion', help='whisk API version', dest='apiversionOverride', metavar='version')
+    parser.add_argument('-i', '--insecure', help='allow untrusted SSL certificates', action='store_true', default=True)
```

Comment:
note default is `-i == insecure` so it is not a breaking change.
further enhancements would set this value relative to api host (as noted in my comment here #428).